packages/repository-tests/src/crud/relations/fixtures/models/order.model.ts

```diff
@@ -1,19 +1,21 @@
 // Copyright IBM Corp. 2019. All Rights Reserved.
-// Node module: @loopback/repository
+// Node module: @loopback/repository-tests
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {belongsTo, Entity, model, property} from '../../..';
+import {belongsTo, Entity, model, property} from '@loopback/repository';
 import {Customer, CustomerWithRelations} from './customer.model';
 import {Shipment, ShipmentWithRelations} from './shipment.model';
+import {mixedIdType} from '../../helpers';
 
 @model()
 export class Order extends Entity {
   @property({
-    type: 'string',
+    type: 'number',
```

Comment:
I think this needs to be `features.idType`?

Reply:
We decided to use `MixedIdType` instead of `features.idType` or `unknown`. 
The reason that we don't want to use `unknown` is that, ' in relations, we need an actual type for a foreign key type' -- cc @jannyHou

Reply:
`unknown` would result in incompatible type error when use `Order`(or `Partial<Order>`) as a type to describe the data. My expected solution is using `features.idType` but I also understand the refactor is not trivial, if `MixedIdType` could get the tests pass in this PR I would vote for it :) We can improve it in another story.

Reply:
Fair enough, I am ok to use `MixedIdType` for now.